### PR TITLE
Updated UI instructions and added light mode

### LINKED ISSUE #815: Document UI path updates for beta 6 style changes
Initial list, probably includes more:
- “Save As” button and other UI elements have been relocated
- Update bucket settings -> Now click on **Settings**
- Time format on heatmap config options

### PR DESCRIPTION
**Pull request by sanderson**
Closes #815 and #832 

- [x] Tests pass (no build errors)
- [x] Rebased/mergeable

### CODE REVIEW

File: content/v2.0/visualize-data/templates/view-template.md [hunk 1]
```diff
@@ -6,34 +6,68 @@ v2.0/tags: [templates]
...
-The following Telegraf-related dashboards templates are available:
+The following Telegraf-related dashboard templates are available:
 
+- [Apache Data](#apache-data)
 - [Docker](#docker)
 - [Getting Started with Flux](#getting-started-with-flux)
+- [GitHub Data](#github-data)
+- [InfluxDB 2.0 OSS Metrics](#influxdb-2-0-oss-metrics)
+- [JMeter](#jmeter)
 - [Kubernetes](#kubernetes)
```
Comment (kelseiv) [inline content/v2.0/visualize-data/templates/view-template.md]: Tim just asked us to update this list today... :0)))

File: content/v2.0/visualize-data/templates/view-template.md [hunk 2]
```diff
@@ -6,34 +6,68 @@ v2.0/tags: [templates]
...
-- [Local Metrics](#local-metrics)
 - [Nginx](#nginx)
 - [Redis](#redis)
 - [System](#system)
 
+### Apache data
+The Apache data dashboard template contains and overview of Apache HTTP server metrics.
```
Comment (kelseiv) [inline content/v2.0/visualize-data/templates/view-template.md]: an overview

File: content/v2.0/visualize-data/templates/view-template.md [hunk 3]
```diff
@@ -81,6 +115,58 @@ This dashboard is designed to get you started with the Flux language. It contain
...
+- Issues
+- Summary
+
+#### Plugins
+- [`github` plugin](/v2.0/reference/telegraf-plugins/#github)
+
+
+### InfluxDB 2.0 OSS Metrics
+The InfluxDB 2.0 OSS Metrics dashboard shows a visual overview of some of the metrics
+available from the Local InfluxDB metrics endpoint located at `/metrics`.
```
Comment (kelseiv) [inline content/v2.0/visualize-data/templates/view-template.md]: is "Local" part of the endpoint name?

File: content/v2.0/visualize-data/templates/view-template.md [hunk 4]
```diff
@@ -81,6 +115,58 @@ This dashboard is designed to get you started with the Flux language. It contain
...
+- # of Scrapers
+- # of Tasks
+- Local Object Store IO
+- Memory Allocations (Bytes)
+- Memory Usage (%)
+- Memory Allocs & Frees (Bytes)
+
+### JMeter
+The JMeter dashboard template provides an overview of JMeter metrics.
+It displays the following:
```
Comment (kelseiv) [inline content/v2.0/visualize-data/templates/view-template.md]: curious does it make sense to use same phrase "the following metrics:" "the following information" "the following:"

File: content/v2.0/visualize-data/visualization-types/graph-single-stat.md [hunk 5]
```diff
@@ -38,8 +38,8 @@ For consistent Single Stat results, the query should return a single table.
 {{% /note %}}
 
 ## Graph + Single Stat Controls
-To view **Graph + Single Stat** controls, click the settings icon ({{< icon "gear" >}})
-next to the visualization dropdown in the upper right.
+To view **Graph + Single Stat** controls, click **{{< icon "gear" >}} Customize** next to
```
Comment (kelseiv) [inline content/v2.0/visualize-data/visualization-types/graph-single-stat.md]: to be sure, for all these different controls, folks click the icon gear and Customize and get different visualizations?

File: content/v2.0/visualize-data/templates/_index.md [hunk 6]
```diff
@@ -1,14 +1,20 @@
...
-    name: Manage templates
+    name: Manage dashboard templates
     parent: Visualize data
-weight: 101
+weight: 104
 ---
 
 Templates are a way to create and share dashboards across organizations without starting from scratch.
 
+### Template types
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/visualize-data/templates/_index.md]: Yes, they are. Static templates are provided by us and can't be changed. User templates are dashboard templates that users have created and saved themselves.

File: content/v2.0/visualize-data/templates/create-template.md [hunk 7]
```diff
@@ -1,17 +1,17 @@
...
   v2_0:
     name: Create a template
-    parent: Manage templates
+    parent: Manage dashboard templates
 weight: 201
 
 ---
 
-## Import a template
+## Import a dashboard template
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/visualize-data/templates/create-template.md]: "Import" is the language the dashboard uses. I moved this to the bottom of the doc and updated it to say "Import an existing dashboard template." This walks through the process of uploading a dashboard template file and saving it as a template.

File: content/v2.0/visualize-data/templates/create-template.md [hunk 8]
```diff
@@ -31,16 +31,25 @@ weight: 201
...
     {{< nav-icon "settings" >}}
 
 2. Select the **Templates** tab.
-3. Hover over the template you want to clone, then click {{< icon "clone" >}}.
-4. Click **Clone** to confirm. The cloned template appears in your list.
+3. Select **User Templates**.
+
+    {{< img-hd src="/img/2-0-templates-type-select.png" alt="Select User Templates" />}}
+
+4. Hover over the template you want to clone and click **{{< icon "clone" >}}**.
```
[the remark this answers is not in the document]
Comment (sanderson) [reply content/v2.0/visualize-data/templates/create-template.md]: Yes, they do.

File: content/v2.0/visualize-data/visualization-types/graph-single-stat.md (hunk 5 above)
Comment (sanderson) [reply content/v2.0/visualize-data/visualization-types/graph-single-stat.md]: This is the button in the UI that users click to open visualization options:

image

**Pull request merged**